selfdrive/car/isotp_parallel_query.py

```diff
@@ -131,16 +134,24 @@ def get_data(self, timeout, total_timeout=None):
             results[tx_addr] = dat[len(expected_response):]
             request_done[tx_addr] = True
         else:
-          request_done[tx_addr] = True
-          cloudlog.warning(f"iso-tp query bad response: 0x{dat.hex()}")
+          error_code = dat[2] if len(dat) > 2 else -1
+          if error_code == 0x78:
+            pending_responses[tx_addr] = time.monotonic()
+            if self.debug:
+              cloudlog.warning(f"iso-tp query response pending: {hex(tx_addr)}")
```

Comment:
```suggestion
              cloudlog.warning(f"iso-tp query response pending: {hex(tx_addr[0])} {hex(tx_addr[1])}")
```

Broke --debug mode, `tx_addr` is an (address, subaddress) tuple.

```
FW query exception: Traceback (most recent call last):            
  File "./fw_versions.py", line 340, in get_fw_versions
    fw_versions[r.rx_offset].update(query.get_data(t))
  File "/data/pythonpath/selfdrive/car/isotp_parallel_query.py", line 141, in get_data                                               
    cloudlog.warning(f"iso-tp query response pending: {hex(tx_addr)}")                                                               TypeError: 'tuple' object cannot be interpreted as an integer
```